security/s2a/internal/crypter/testutil/wycheproofutil.go

```diff
@@ -0,0 +1,54 @@
+package testutil
+
+import (
+	"encoding/json"
+	"os"
+	"testing"
+)
+
+// ParseWycheProofTestVectors takes a path to a WycheProof test vector, a test
+// group filter, and returns the resulting CryptoTestVector. The test group
+// filter will be used to filter out unsupported test inputs.
+func ParseWycheProofTestVectors(
```

Comment:
Got it, I was debating on using a dot import here or not and I initially decided to use it since otherwise we'll have to write `testutil` all across the test. I updated it to use the fully qualified name now.

Reply:
SG, thanks for the fix!